source/agora/node/Ledger.d

```diff
@@ -221,16 +240,19 @@ public class Ledger
 
     public Hash[] getMerklePath (ulong block_height, Hash hash) @safe
     {
-        if (block_height >= this.ledger.length)
-            return null;
-
-        const(Block)* block = &this.ledger[block_height];
-
-        size_t index = block.findHashIndex(hash);
-        if (index < block.txs.length)
-            return block.getMerklePath(index);
+        Block block;
```

Comment:
If the wanted block height is greater than our `this.block_height`, then this function should return null.

Also, add an assertion like before so `block.getMerklePath()` must return true.

Reply:
Applied the suggestion